docs/Getting Started/importRequireGlobalVars.md

```diff
@@ -0,0 +1,77 @@
+# IMPORT, REQUIRE and Global Namespaces
+
+Depending on how the `@hpcc-js` packages have been included into your web app, there are 3 common paradigms for accessing the widgets within the packages.  While the majority of the examples in this documentation use the `import` method, it is important to note that depending on your apps configuration any of the 3 common options are interchangeable.
+
+##  \<SCRIPT> and Global Namespaces
+
+Tends to be used on smaller or "quick" web page construction, the packages are typically loaded from a CDN server and are accessed via global namespace.  Each `@hpcc-js` package will set up a global variable matching the package name.
+**Note:**  Because the global variable includes "/" in its name, it will need to be accessed using `OBJ["prop"]` syntax, this is deliberate as it reduces the chance that our global variable names will clash with other third party libraries.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>CDN + SCRIPT</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="css/index.css">
+    <script src="https://unpkg.com/@hpcc-js/util"></script>
+    <script src="https://unpkg.com/@hpcc-js/common"></script>
+    <script src="https://unpkg.com/@hpcc-js/api"></script>
+    <script src="https://unpkg.com/@hpcc-js/chart"></script>
+    <script>
+        var hpccChart = window["@hpcc-js/chart"];
+    </script>
+</head>
+
+<body onresize="doResize()">
+    <div id="placeholder">
+    </div>
+    <script>
+        const myPie = new hpccChart.Pie();
+        ...
```

Comment:
What does this do?

Reply:
It may be better to simply add a target, columns and data to prevent any potential confusion.